### DISCUSSION TITLE
Can you provide the "@nx._dispatch" decorator to as many algorithms as possible?

### DISCUSSION THREAD

**Discussion by MikuSugar** [networkx/networkx#6793] [General]
Hello, I am developing a computational backend plugin for NetworkX. I noticed that there are many algorithms that haven't been decorated with "@nx._dispatch". Could you please add the "@nx._dispatch" decorator to as many algorithms as possible? 🙏

**Comment by MridulS** [comment]
Thanks for the post :) https://github.com/networkx/networkx/pull/6688 should be in soon which does cover most of the algorithms.

But I am very interested in what kind of backend you are working on?

**Reply by MikuSugar** [reply]
Thank you for your response. The project I am working on is a company project, so I cannot disclose too many details. NetworkX is highly popular, and therefore, we are planning to leverage its plugins to allow users to perform large-scale graph computations using a familiar API.
